modules/vulnerabilities/unix/irc/unrealirc_3281_backdoor/templates/overshare.erb

```diff
@@ -0,0 +1,2 @@
+<% require 'json' -%>
+<%= JSON.parse(@json_inputs)["strings_to_leak"].join("\n----\n") %>
```

Comment:
There is no reason the strings can't all end up in separate files -- the NFS example just dumped them into a file as a simple example